Makefile.gmloader

```diff
@@ -50,7 +60,7 @@ GENERIC_OBJ:=\
 
 OBJ=$(patsubst build/%,build/${ARCH}/%,${GENERIC_OBJ})
 
-all: build/${ARCH}/gmloader/gmloader
+all: build/${ARCH}/gmloader/gmloadernext.${DEVICE_ARCH}
```

Comment:
Wondering if we should drop the next part and just use gmloader as Johnny originally intended this to replace the old gmloader.

Reply:
I don't mind at all removing `next`. Let's ask Johnny :)

Reply:
@JohnnyonFlame What do you think? @cdeletre is it possible to separate gamedata from savedata in this PR as well, or do you think that would warrant a separate pull request?